x-pack/plugins/observability_shared/public/plugin.ts

```diff
@@ -57,6 +57,7 @@ export class ObservabilitySharedPlugin implements Plugin {
     return {
       navigation: {
         PageTemplate,
+        registerSections: this.navigationRegistry.registerSections,
```

Comment:
Why do we need to add this?

Reply:
because this is also used in plugin start method of synthetics. This simplifies so that it can be used when setup is done and it depends on start service from a plugin contract.